build-docker-image.sh

```diff
@@ -18,6 +18,20 @@ run() {
     fi
   fi
 
+  if grep -i centos "${dockerfile}" >/dev/null 2>/dev/null; then
+      # build debian sysroot in a separate ubuntu container
+      # (only done for x86-linux-gnu ATM)
+      rm -rf qemu
+      mkdir qemu
+      cp linux-image.sh qemu/
+      docker run \
```

Comment:
Can we instead use the multi-step builder pattern in the`Dockerfile` for this?

Reply:
ok, switched from this to a multi-stage build in the x86 dockerfile